docs/RELEASING.md

```diff
@@ -1,5 +1,5 @@
 ## Releasing npm package
 
-* [Create a Jira release ticket](https://jira.sonarsource.com/projects/REL)
+- [Create a Jira release ticket](https://jira.sonarsource.com/projects/REL) like [this one](https://sonarsource.atlassian.net/browse/REL-2313)
```

Comment:
Why change from `*` to `-`? It's breaking the Markdown [here](https://github.com/SonarSource/eslint-plugin-sonarjs/blob/9777e71e971141aa6714af3a0daf3ff8bf006732/docs/RELEASING.md).

Reply:
I would like to blame my editor for this.

Reply:
Let's blame it then ;)